webrtc_videoroom/lib/videoroom/room.ex

```diff
@@ -20,15 +22,25 @@ defmodule Videoroom.Room do
   def init(room_id) do
     Membrane.Logger.info("Spawning room proces: #{inspect(self())}")
 
+    # when running via docker we want to listen at 0.0.0.0
+    # but our packets still need valid IP address in their
+    # headers. We store it under `mock_ip`.
```

Comment:
When docker is running with `--network=host`, it is unnecessary to listen on 0.0.0.0, so consider changing this comment